src/App.js

```diff
@@ -1,20 +1,441 @@
 import React from "react";
```

Comment:
I think this file is bloated to the maximum. We could do well with making it a lot leaner. I need to scroll almost 400 lines of code until I see what the output here is.